rust/stackable-cockpit/src/platform/cluster/mod.rs

```diff
@@ -9,13 +9,13 @@ pub use resource_request::*;
 
 #[derive(Debug, Snafu)]
 pub enum ClusterError {
-    #[snafu(display("failed to parse node cpu"))]
-    ParseNodeCpu {
+    #[snafu(display("failed to parse node cpu quantity"))]
+    ParseCpuQuantityError {
```

Comment:
1. The old rust name was more helpful than the new one
2. You don't need to suffix every variant with `Error`, we already know that it's in the `Error` enum

Reply:
Changed in ac8ed84